build.gradle

```diff
@@ -224,10 +224,51 @@ configurations.all {
         force "io.netty:netty-common:${versions.netty}"
         force "io.netty:netty-handler:${versions.netty}"
         force "io.netty:netty-transport:${versions.netty}"
-        force "io.netty:netty-transport-native-unix-common:${versions.netty}"
+        }
+}
+
+//create source set 'newTest'
+//add classes from the main source set to the compilation and runtime classpaths of the newTest
+sourceSets {
+    newTest {
+        compileClasspath += sourceSets.main.output
+        runtimeClasspath += sourceSets.main.output
     }
 }
 
+//add new task that runs new tests
```

Comment:
How does this impact test results/code coverage information from the CI workflow?  I'd like to change a little as possible for the standard workflows, but an _extra_ task to run specific tests could be useful

Reply:
There is a task called `integrationTest` (renamed from newTest) that runs our new tests.
It should not impact code coverage, by default jacoco generates a report based on the /build/jacoco/test.exec file only